en/components/rating.md

```diff
@@ -0,0 +1,89 @@
+| 🚫 componentVersion | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| 🚫 metadata         | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it. |
+| Label              | Text layer that can be changed or hidden if not needed                                                                                                              |
+| Value              | Contains all the rating icons                                                                                                                 |
+
+After detaching, you may increase the number of the rating icons by duplicating any of the existing ones nested in the `Value` frame, and the size will be adjusted thanks to the Auto Layout.
+
+To decrease the number of the rating icons, there is no need to detach the component, you should hide the not needed one from the Layers panel.
+
+## Using the Rating in Sketch
+
+The Raiting component in Sketch has almost the same building elements like in Figma. There are Label and Value and the component comes with five rating icons by default. In case that there is need for more rating icons, the component needs to be detached by right-clicking on the component and selecting `Detach from Symbol` from the contextual menu. After detaching you should see the following detached layers:
+
+| Layer                        | Use                                                                                                                                                  |
+| ---------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------- |
+| 🚫 metadata        | A special locked layer starting with a prohibited icon. This layer is required by the code generation and you should avoid deleting or modifying it.  |
+| Label              | Text layer that can be changed or hidden if not needed                                                                                                                                                      |
+| Value              | Contains all the rating icons                                                                                                                         |
+
+In order to increase or decrese the number of rating icons, you should use the `Detach from Sybmol` option again over the `Value` symbol and then by duplicating or deleting any of the rating icons that already exist you will be able to achive the desired amount of icons.
+
+## Using the Rating in Adobe XD
+
+> [!NOTE]
+> WIP: Currently, we don't support the Rating component in Adobe XD.
+
+## Interaction States
+
+The Rating can be inserted in an enabled or disabled state.
```

Comment:
In Sketch the Rating can be inserted in an enabled or disabled state, while in Figma when inserted the Rating component is always enabled by default, and in order to disable it you can do that from the property panel.

Reply:
Fixed

Reply:
@AnjiManova it should actually be properties panel, not property, Vasil misled you 😃